tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2879,6 +2891,10 @@ describe('scalar card', () => {
           MIN_VALUE: 4.532374100719424,
           MAX_VALUE: 18.976711362032464,
           PERCENTAGE_CHANGE: 1.2427022518765645,
+          STEP_AT_MAX: 4,
```

Comment:
AFAICT there is no test where STEP_AT_MAX is anything other than the last step. Can we add a test case that gives a little more confidence that it is working?

Reply:
Similarly, AFAICT, there is no test where range does not include the max step. Could you add a case for that, too?

Reply:
Added.